SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Security Policy
+
+## Supported Versions
```

Comment:
Do we document this elsewhere in the repo? If so, then I think we should not duplicate it, since that means that we need to remember to change it in two places.

Reply:
I think that this is the only place in the repo.